protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -4188,6 +4196,60 @@ message QuantumRewardsPerEpoch {
   string total_quantum_rewards = 2;
 }
 
+// Request to list all team members' statistics.
+message ListTeamMembersStatisticsRequest {
+  // Restrict team statistics to those with the given team ID.
+  string team_id = 1;
+  // Restrict team members' statistics to those with the given party ID.
+  optional string party_id = 2;
+  // Defines the number of past epochs to aggregate data from. By default, it takes
+  // the last 10 epochs.
```

Comment:
This says 30 epochs in the GQL docs. Are they actually different?

Reply:
It should be 10. It's changed.